Altis_Life.Altis/core/functions/TON/fn_sendMessage.sqf

```diff
@@ -0,0 +1,69 @@
+/*
+    File: fn_sendMessage.sqf
+    Author: blackfisch
+
+    Description: send message to target
+*/
+params [
+    ["_ehParams", [], [[]]],
+    ["_target", "player", [""]]
+];
+_ehParams params [
+    ["_control", controlNull, [controlNull]]
+];
+
+_control ctrlEnable false;
+
+private _to = call compile format ["%1",(lbData[3004,(lbCurSel 3004)])];
```

Comment:
Perhaps we can just avoid the call compile here

Reply:
see #613 - wanted to do that for the whole mission at once